site/content/blog/kctrl-app-commands-blog.md

```diff
@@ -0,0 +1,509 @@
+---
+title: "Getting to know App resources better with kctrl"
+slug: kcrl-app-commands-blog
+date: 2022-04-28
+author: Soumik Majumder
+excerpt: "Get better visibility into your app resources with kctrl's new app commands and enhancements to the package commands"
+image: /img/kapp-controller.svg
+tags: ['carvel', 'kapp-controller', 'gitops', 'apps', 'kctrl', 'cli']
+---
+
+Since the previous release of `kctrl`, we have been iterating over how we can help users take a closer look at what App CRs are doing on the cluster. This includes ones authored by users and those that are created as a result of package installations.
+
+As promised, the latest release of `kctrl` introduces a set of commands which allow users to observe and interact with App CRs more conveniently. `kctrl` v0.36.0 also introduces an improved status tailing experience which surfaces relevant information while waiting for resources to reconcile.
```

Comment:
nit: `kctrl` v0.36.0 also improves on status tailing experience

Reply:
I would say it does not improve on it because it was not there before. 
I guess I could get rid of improved all together if that makes more sense 🤔

Reply:
I thought it was there. Yup, if it was not there, we should remove improve.